test/features/caseView.feature

```diff
@@ -27,5 +25,14 @@ Feature: Set of scenarios testing case view functionality
       | Text Field 3        |
       | Text Field 4        |
 
+  @functional
   Scenario: print button visible on case view
-   Then the print button will be visible
+    When a case type containing every field type exists
```

Comment:
Lets make this line more readable from a BDD point of view `Given a case type with the print button configured exist` you can then use(copy) the same implementation from the step 'a case type containing every field type exists' you will find the code in definitionFileSteps.js or better still put it in a new function and call it in both steps

Reply:
Done.